arrow-core-data/src/main/kotlin/arrow/core/Option.kt

```diff
@@ -628,7 +628,7 @@ sealed class Option<out A> : OptionOf<A> {
   fun <B> filterMap(f: (A) -> Option<B>): Option<B> =
     flatMap(f)
 
-  fun <B> filterMap(f: (A) -> B?): Option<B> =
+  inline fun <B> filterMap(f: (A) -> B?): Option<B> =
```

Comment:
Duplicated with `mapNotNull` should we keep one or the other? I think on `Iterable<A>`/Collections we've deprecated `filterMap` in favor of `mapNotNull` which for those types comes from Kotlin's Std.